pyproject.toml

```diff
@@ -33,12 +37,11 @@ extend-ignore = [
     "D408",
     "D409",
     "D413",
+    "E501",
 ]
-ignore = ["E501", "D107"]
-extend-exclude = ["__pycache__", "*.egg_info"]
 per-file-ignores = {"tests/*" = ["D100","D101","D102","D103","D104"]}
 
-[tool.ruff.mccabe]
+[tool.ruff.lint.mccabe]
```

Comment:
All these changes are due to deprecation warnings in `ruff`.

They're drive by changes.